<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_shooting_gun</class>
 <widget class="QDialog" name="dlg_shooting_gun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(57, 169, 255);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Vzdálenost střelby</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CB_Shooting_Distance">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Typ střelby</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CB_Shooting_Type">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Střílená zbraň</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CB_Shooting_Gun">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Vystřeleno munice</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Shooting_AmmoUsed">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Label_Shooting_ID">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string>Potvrdit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
